Documentation/contributing.rst

```diff
@@ -33,8 +33,6 @@ contribute to Cilium:
 +----------------------------------------------------------------------------------+--------------------------+-------------------------------------------------------------------------------+
 | `dep <https://github.com/golang/dep/>`_                                          | >= v0.4.1                | ``curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh``  |
 +----------------------------------------------------------------------------------+--------------------------+-------------------------------------------------------------------------------+
-| `go-bindata <https://github.com/cilium/go-bindata>`_                             | ``a0ff2567cfb``          | ``go get -u github.com/cilium/go-bindata/...``                                |
-+----------------------------------------------------------------------------------+--------------------------+-------------------------------------------------------------------------------+
```

Comment:
Unless the goal is to hide the fact that we're using go-bindata it is still required. This list isn't just what to install, it's a quick way to get a sense for the bigger external dependencies. On the other hand, the .lock file does that more comprehensively so I guess it's ok not to mention it iff we continue to use go run in the makefile.